### ISSUE TITLE
How to determine dataset output XML file name

### ISSUE THREAD

**Issue by emiliom** [NOAA-PMEL/uafnciso#17]
I'm using stand-alone nciso (2.3.6) to generate ISO metadata XML's from 4 datasets on 3 servers -- 2 THREDDS + 1 Hyrax.

For two of those servers (including the Hyrax one), I can see where the dataset output XML file name comes from:
- http://amb6400b.stccmop.org:8080/thredds/forecast_model_data.xml > `thredds_dodsC_model_data_forecast.xml`: from a combination of `service[@base]` and `dataset[@urlPath]`
- http://ingria.coas.oregonstate.edu/opendap/aggregated/catalog.xml > `opendap_hyrax_aggregated_ocean_time_aggregation.xml`: From `thredds:dataset[@urlPath]`

But the two datasets on the third server generate the same output XML file name, `wilson.xml`. The dataset url's are https://wilson.coas.oregonstate.edu/thredds/catalog.xml?dataset=OCOS and https://wilson.coas.oregonstate.edu/thredds/catalog.xml?dataset=LiveOcean. "wilson" is the name of the server and appears in the domain name, but the string doesn't appear at all in the catalog XML entries, let alone in `dataset[@urlPath]`. Is there anything we can tweak in the catalog entries to control the output file names? The only thing that seems different to these two datasets is that nciso is fed a url that include the `dataset` parameter.

Command used, with `java-8-openjdk-amd64`:
```bash
java -Xms1024m -Xmx1024m -jar ncISO-2.3.6.jar -ts https://wilson.coas.oregonstate.edu/thredds/catalog.xml?dataset=LiveOcean -num 1 -depth 20 -iso true -waf ./tests
```

cc @mwengren, @crisien

**Comment by noaaroland** [comment]
Fixed in https://github.com/NOAA-PMEL/uafnciso/commit/16023217bdc6f2b617f13127da13a0ac7b2e3372 and released in 2.3.7.

**Comment by emiliom** [comment]
Thanks!
